src/db/index.ts

```diff
@@ -19,7 +23,7 @@ class DBClient {
     return DBClient.instance
   }
 
-  getDrizzle(): PostgresJsDatabase {
+  getDrizzle(): PostgresJsDatabase<typeof schema> {
```

Comment:
Let's abstract implementation details (optional)
```suggestion
  getDb / getInstance...
```